csrc/python_frontend/python_bindings.cpp

```diff
@@ -472,11 +472,11 @@ void initNvFuserPythonBindings(PyObject* module) {
                 !self.completed(),
                 "Attempting to add to a completed definition!");
             Scalar out = self.defineScalar();
+            auto record_type = (dtype == PrimDataType::Double)
```

Comment:
I thought we are blindly using double for constant/scalars.

Reply:
ok, looks like the record_type is only used by `serde`. I got confused by the variable name and thought we are also dealing with it on the fusion record side.

I'm still a little bit confused why we need to differentiate double/float for scalar on serialization/deserialization, since our fusion record doesn't do that.